testsuite/do_tests.sh.in

```diff
@@ -354,6 +354,7 @@ run_test ()
 # cmake during configuration.
 export PYTHONPATH="${NEST_PYTHONPATH}:${PYTHONPATH}"
 export PATH="${NEST_PATH}:${PATH}"
+export LD_LIBRARY_PATH="${NEST_LD_LIBRARY_PATH}:${LD_LIBRARY_PATH}"
```

Comment:
On macOS (`if test "x${INFO_OS}" = xDarwin ; then`) the library path environment variable is called `DYLD_LIBRARY_PATH`.